servers/quarkus-server/src/main/resources/jni-config.json

```diff
@@ -0,0 +1,46 @@
+[
```

Comment:
same here, it's not clear why this is needed

Reply:
Not seen my comment above??